### PR TITLE
Fixes #14953 - update LICENSE file

### PR DESCRIPTION
**Pull request by tbrisker**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -22,39 +22,19 @@ If not, see [GNU licenses](http://www.gnu.org/licenses/).
...
-  (c) 2008-2012 Allan Jardine
-  (c) 2010 Xuan Luo
-  (c) 2006-2009 Mika Tuupola, Dylan Verheul
-  (c) 2010 Giva, Inc.
-  (c) 2012-2014 Jon Nylander
+* app/views/unattended/ztp/provision.erb is (c) 2013, Juniper Networks under 2-clause BSD license.
+* public/blank.html is (c) 2005, Brad Neuberg, bkn3@columbia.edu under MIT license.
+* lib/tasks/convert.rake is (c) 2008, Matson Systems, Inc. under Revised BSD license.
+* extras/noVNC/websockify is (c) 2011, Joel Martin under LGPL v3 license.
+* vendor/assets/fonts/ is (c) 2015, Red Hat Inc. under SIL Open Font License v1.1 and LGPL v2.1 licenses.
```
Comment (domcleal) [inline LICENSE]: 1. vendor/assets/fonts/overpass\* I think?
2. what's the licence of vendor/assets/fonts/AllBullets*?
3. SIL _or_ LGPL
4. 2011, according to the files
5. ~~Missing the text of the SIL licence in this file.~~ it's there

File: LICENSE [hunk 2]
```diff
@@ -22,39 +22,19 @@ If not, see [GNU licenses](http://www.gnu.org/licenses/).
...
+* vendor/assets/javascripts/noVNC/ is (c) 1999 AT&T Laboratories Cambridge, (c) 1996 Widget Workshop, Inc., (c) 1996 Jef Poskanzer <jef@acme.com>, (c) 2011 Joel Martin, (c) Hiroshi Ichikawa <http://gimite.net/en/>, (c) Martijn Pieters <mj@digicool.com>, Samuel Sieb <samuel@sieb.net> under MPL 1.1, LGPL v3, GPL v2+, MIT and BSD licenses.
```
Comment (domcleal) [inline LICENSE]: I think GPL v2+ is an option, so `(MPL 1.1 or GPL v2+ or LGPL v2.1+) and LGPL v3 and LGPL v2+ and MIT and 3-clause BSD`

Comment (domcleal) [inline LICENSE]: Correction, 3-clause should be "Revised" for consistency in this file.

File: LICENSE [hunk 3]
```diff
@@ -22,39 +22,19 @@ If not, see [GNU licenses](http://www.gnu.org/licenses/).
...
+* vendor/assets/javascripts/diff.js is (c) 2009-2011, Kevin Decker <kpdecker@gmail.com> under Revised BSD license.
+* vendor/assets/javascripts/jquery.cookie.js is (c) 2006, 2014 Klaus Hartl (stilbuero.de) under MIT license.
+* vendor/assets/javascripts/jquery.dataTables.min.js is (c) 2008-2012 Allan Jardine under GNU GPL v2 and BSD licenses.
```
Comment (domcleal) [inline LICENSE]: prefer `or` over `and` to indicate dual licensing

File: LICENSE [hunk 4]
```diff
@@ -22,39 +22,19 @@ If not, see [GNU licenses](http://www.gnu.org/licenses/).
...
+* vendor/assets/javascripts/jquery.flot.axislabels.js is (c) 2010 Xuan Luo under GNU GPL v3 and MIT licenses.
```
Comment (domcleal) [inline LICENSE]: currently only under MIT

File: LICENSE [hunk 5]
```diff
@@ -1,7 +1,7 @@
 The Foreman repository/package is licensed under the GNU GPL v3 or newer,
 with some exceptions.
 
-Copyright (c) 2009-2015 to Ohad Levy, Paul Kelly and their respective owners
+Copyright (c) 2009-2015 to Ohad Levy, Paul Kelly and their respective owners.
```
Comment (domcleal) [inline LICENSE]: 2016

File: LICENSE (hunk 1 above)
Comment (tbrisker) [inline LICENSE]: 1+2. they are the same license and copyright, I created AllBullets to allow multi-line text areas to be hidden (https://github.com/tbrisker/AllBullets).
3. fixed
4. I'll make it 2011-2016.

File: LICENSE (hunk 2 above)
Comment (tbrisker) [inline LICENSE]: fixed. I wonder if there is a license they _didn't_ use for this library...

File: LICENSE (hunk 3 above)
Comment (tbrisker) [inline LICENSE]: fixed

File: LICENSE (hunk 4 above)
Comment (tbrisker) [inline LICENSE]: fixed

**Comment by tbrisker** [comment]
@domcleal fixed all your comments. IANAL, but I hope everything is in order now.

**Comment by domcleal** [comment]
Merged as 5a755fe, thanks @tbrisker.

**Pull request closed without merging**